ui/store/actions.js

```diff
@@ -399,7 +431,7 @@ export function forgetDevice(deviceName) {
   return async (dispatch) => {
     dispatch(showLoadingIndication());
     try {
-      await promisifiedBackground.forgetDevice(deviceName);
+      await submitRequestToBackground('forgetDevice', [deviceName]);
     } catch (error) {
```

Comment:
We wil replace all calls to `await promisifiedBackground.forgetDevice` by `await submitRequestToBackground`.